.travis.yml

```diff
@@ -1,8 +1,4 @@
 language: java
 
 jdk:
-  - oraclejdk8
-  - oraclejdk9
-  - openjdk10
-  - openjdk11
-  - openjdk-ea
+  - openjdk8
```

Comment:
is `openjdk8` the only version that can pass tests? -- java 8 is a bit old. Probably we want to fix the test for later version of jdk?

Reply:
added jdk10 and jdk11 as well.